front/src/modules/ui/table/types/ViewField.ts

```diff
@@ -91,6 +91,7 @@ export type ViewFieldDefinition<T extends ViewFieldMetadata | unknown> = {
   columnOrder: number;
   columnIcon?: JSX.Element;
   filterIcon?: JSX.Element;
+  isVisible?: boolean;
```

Comment:
I see that you introduce a views component folder. Reflexion for later: should we separate view from tables (regarding this type or useLoadView hook). This would mean that the table does not direclty know about viewField which is business logic. And would instead only know about a TableConfiguration and a mapping would be done when instanciating Tables and ViewFields.

We can also decide that tables and kanbna we have a dependency on ViewFields (they are aware of it). In this case, we just move viewFields components to views and leave table code as it is. I would advocate to follow this path for now. It means that we cannot make table without viewFields which is unlikely to happen in our codebase, we can do the work to abstract it if we want to make table a agnostic package that open-source community can use for other purposes.

Same question goes for filter-n-sort actually

Right now, it seems that both are coupled.